buildscripts/condarecipe.local/meta.yaml

```
@@ -30,18 +30,18 @@ requirements:
     - llvm-openmp              # [osx]
   host:
     - python
-    - numpy
+    - numpy {{ numpy }}
```

this is not needed anymore
```suggestion
    - numpy
```